src/modules/Builder/components/ConditionRow/Condition/Condition.tsx

```diff
@@ -17,6 +25,138 @@ import {
   getStateOptions,
 } from './Condition.utils';
 
+export const SwitchCondition = ({
```

Comment:
Maybe it's better to create its own component for SwitchCondition? This will make Condition.tsx not so big.